### PR TITLE
[MM-20290] Add unit tests to the "ShowRoleCmd" mmctl command

### PR DESCRIPTION
**Pull request by joshuabezaleel**
Summary: 
Add unit tests to the "ShowRoleCmd" mmctl command

Jira: https://mattermost.atlassian.net/browse/MM-20290
Fixes: https://github.com/mattermost/mattermost-server/issues/13403

### CODE REVIEW

**Comment by joshuabezaleel** [comment]
The lint failed and yield an error of 
`commands/permissions_test.go:9: File is not `goimports`-ed with -local github.com/mattermost/mmctl (goimports)
	"github.com/mattermost/mattermost-server/v5/model"`

Any idea on how to fix this? :/

File: commands/permissions_test.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 
 	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
```
Comment (isacikgoz) [inline commands/permissions_test.go]:
```suggestion
﻿﻿﻿	"github.com/mattermost/mmctl/printer"

	"github.com/mattermost/mattermost-server/v5/model"
```

**Comment by joshuabezaleel** [comment]
Hi @mgdelacroix , truly sorry that I missed this since there has been some personal issue here and I just got the chance to open my laptop :disappointed: So sorry for this, I will definitely respond much faster in the future. 

Thank you very much for putting the fix up for me!!! Truly appreciate it :slightly_smiling_face:

**Comment by mgdelacroix** [comment]
No worries @joshuabezaleel! Only a quick fix was needed. Hope that everything is alright :)

**Pull request merged**